spectator-ext-ipc/src/main/java/com/netflix/spectator/ipc/http/HttpRequestBuilder.java

```diff
@@ -291,9 +291,10 @@ public HttpResponse send() throws IOException {
         }
       } catch (IOException e) {
         if (attempt == numAttempts || !retryPolicy.shouldRetry(method, e)) {
+          LOGGER.warn("attempt {} of {} failed: {} {}", attempt, numAttempts, method, uri);
```

Comment:
At this point the exception is bubbled up to the caller to log or otherwise handle with additional context. I don't think we should be adding what is likely a duplicate warning here.

Reply:
Fair. Removed.